charts/stable/unifi/values.yaml

```
@@ -34,7 +34,28 @@ service:
         enabled: true
         port: 3478
         targetPort: 3478
-        protocol: UDP
+        protocol: UDP  
```

```suggestion
        protocol: UDP
```
Fix the trailing spaces error